rules/tines_rules/tines_global_resource_destruction.yml

```diff
@@ -0,0 +1,41 @@
+AnalysisType: rule
+DisplayName: "Tines Global Resource Destruction"
+Enabled: true
+Filename: tines_global_resource_destruction.py
```

Comment:
please add rule description, runbook, and reference documentation that may be included in the scoped doc

Reply:
https://www.notion.so/pantherlabs/Hackathon-Tines-GlobalResource-Destruction-ef9768d7853e421ca045faecd6f79aab?pvs=4

Unfortunately no metadata for me to all from the spec. I can add a Description just re-stating the rule logic, but I will need guidance for runbook and reference docs

Reply:
description: A tines user has destroyed a global resource 
runbook: Possible data destruction. Please reach out to the tines user and confirm this was done for valid business reasons.